app/views/admin/tag_sets/show.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<% add :menu, "All Tag Sets" => admin_tag_sets_path(@asset) %>
+
+<%= page_title('Tag Set', @tag_set.name) %>
+
+<dl>
+  <dt>Name</dt><dd><%= @tag_set.name %></dd>
+  <dt>Tag group (i7)</dt><dd><%= link_to @tag_set.tag_group.name, @tag_set.tag_group %></dd>
+  <dt>Tag group (i5)</dt><dd><%= @tag_set.tag2_group ? (link_to @tag_set.tag2_group&.name, @tag_set.tag2_group) : 'None'%></dd>
+</dl>
+
+<br>
+<% if can? :create, TagLayoutTemplate %>
+  <%= link_to 'Create a new tag layout template from this tag set', new_tag_layout_template_path(tag_group_id: @tag_set.tag_group.id, tag2_group_id: @tag_set.tag2_group&.id) %>
+<br>
```

Comment:
I have added this to criteria to make creating tag layout templates automatically by selecting the correct tag groups in the target page. Do we have a sidebar link to access the readonly view of tag sets? Note that this is different from managing (creating) tag sets.

Reply:
Same reply as below, added a link on home page and made show and index non-admin.